tests/org.eclipse.swt.tests/JUnit Tests/org/eclipse/swt/tests/junit/Test_org_eclipse_swt_browser_Browser.java

```diff
@@ -518,6 +519,98 @@ public void changing(LocationEvent event) {
 	for (int i = 0; i < 100; i++) browser.removeLocationListener(listener);
 }
 
+@Test
+public void test_setUrlAfterDisposed() {
+	Browser testBrowser = createBrowser(shell, SWT.NONE);
+	testBrowser.dispose();
+	assertThrows(SWTException.class, () -> testBrowser.setUrl("about:blank"));
+}
+
+@Test
+public void test_isSetUrlNotCustomTextUrlAfterSetText() {
```

Comment:
Since all the added tests are dedicated to test behavior of `isLocationForCustomText()`, it would be good to have the test names always start with the tested method and append the use case to that. E.g., `test_isLocationForCustomText_falseAfterSetUrl`.

Reply:
Done